From bd51dbd5a312cdf1f39e115d8c95e8f641a43975 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:24 -0400
Subject: [redhat] kabi: Adding symbol add_wait_queue_exclusive (kernel/wait.c)

Message-id: <76743a8bcefec23154f9730c489d347ea968a698.1410510773.git.poros@redhat.com>
Patchwork-id: 95911
O-Subject: [PATCH RHEL7.1 024/233] kabi: Adding symbol add_wait_queue_exclusive (kernel/wait.c)
Bugzilla: 1071097
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Kernel thread wait for event, To wait on a cv in the notification context.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/add_wait_queue_exclusive
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/add_wait_queue_exclusive
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/add_wait_queue_exclusive

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/add_wait_queue_exclusive b/redhat/kabi/kabi-module/kabi_ppc64/add_wait_queue_exclusive
new file mode 100644
index 0000000..b3aebd9
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/add_wait_queue_exclusive
@@ -0,0 +1,2 @@
+#1-
+0x75994700 add_wait_queue_exclusive vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/add_wait_queue_exclusive b/redhat/kabi/kabi-module/kabi_s390x/add_wait_queue_exclusive
new file mode 100644
index 0000000..07c09c5
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/add_wait_queue_exclusive
@@ -0,0 +1,2 @@
+#1-
+0xd4d250e2 add_wait_queue_exclusive vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/add_wait_queue_exclusive b/redhat/kabi/kabi-module/kabi_x86_64/add_wait_queue_exclusive
new file mode 100644
index 0000000..0890cde
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/add_wait_queue_exclusive
@@ -0,0 +1,2 @@
+#1-
+0xee46885d add_wait_queue_exclusive vmlinux EXPORT_SYMBOL
-- 
1.7.1